examples/with-imagekit/next.config.mjs

```diff
@@ -0,0 +1,15 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  reactStrictMode: true,
+  images: {
+    remotePatterns: [
+      {
+        protocol: "https",
+        hostname: "ik.imagekit.io",
```

Comment:
Should we add a comment here with link to our URL endpoints documentation?

Reply:
Done, added a comment